application/backend/src/main/java/pl/ds/bulma/components/models/ImageComponent.java

```diff
@@ -67,4 +68,8 @@ public String getSrc() {
   public String getAssetReference() {
     return LinkUtil.handleLink(assetReference, resource.getResourceResolver());
   }
+
+  public boolean getIsVideo() {
+    return Stream.of("mp4", "ogg").anyMatch(getSrc()::endsWith);
```

Comment:
Did you try `guessContentTypeFromName`? I think it could find more types?

```
String mimeType = URLConnection.guessContentTypeFromName(path);
return mimeType != null && mimeType.startsWith("video");
```

Reply:
Fixed.